o2a/converter/workflow.py

```diff
@@ -55,6 +57,15 @@ def __init__(
             "from airflow.utils.trigger_rule import TriggerRule",
             "from airflow.utils import dates",
         }
+        self.library_folder = os.path.join(self.input_directory_path, HDFS_FOLDER, LIB_FOLDER)
+        self.jar_files = self.get_lib_files(extension=".jar")
+
+    def get_lib_files(self, extension: str) -> List[str]:
```

Comment:
I'm not sure if this method should be in this class. This code does not seem to me related to this object. I think it is worth looking for a better place for him.

Reply:
Any proposal ?

Reply:
Seperate file.

Reply:
Resolving it as we have discussion about it above.